kiwmi/desktop/output.c

```diff
@@ -376,8 +386,129 @@ new_output_notify(struct wl_listener *listener, void *data)
     wl_signal_emit(&desktop->events.new_output, output);
 }
 
+void
+output_layout_change_notify(struct wl_listener *listener, void *UNUSED(data))
+{
+    struct wlr_output_configuration_v1 *configuration =
+        wlr_output_configuration_v1_create();
+    struct kiwmi_desktop *desktop =
+        wl_container_of(listener, desktop, output_layout_change);
```

Comment:
Would feel more natural to me to swap these and first get the `desktop`, since it’s basically converting the parameter to a type that we can use.